scripts/west_commands/twister_cmd.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2022 Google
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+
+from west import log
+
+from zephyr_ext_common import Forceable
+from zephyr_ext_common import get_twister_path
+
+# Resolve path to twister libs and add imports
+twister_path = get_twister_path()
+os.environ["ZEPHYR_BASE"] = str(twister_path.parent)
+
+sys.path.insert(0, str(twister_path))
+sys.path.insert(0, str(twister_path / "pylib" / "twister"))
+
+from importlib.util import spec_from_loader, module_from_spec
+from importlib.machinery import SourceFileLoader
+
+spec = spec_from_loader(
+    "twister", SourceFileLoader("twister", str(twister_path / "twister"))
+)
+twister = module_from_spec(spec)
+spec.loader.exec_module(twister)
+
+from twisterlib.environment import add_parse_arguments, parse_arguments
+
+BUILD_DESCRIPTION = """\
+Convenience wrapper for twister.
+"""
+
+class Twister(Forceable):
```

Comment:
I went back and double checked since you said you were using `west build` as inspiration. I'm guessing that's why this line is here. Inheriting from `Forceable` is not necessary for `west twister` because you do not ever call `check_force`. Just inherit from `WestCommand` instead, as recommended in the documentation:

 https://docs.zephyrproject.org/latest/develop/west/extensions.html#adding-a-west-extension.